src/BinSkim.Sdk/SdkResources.resx

```diff
@@ -240,4 +240,7 @@
   <data name="MetadataCondition_ImageIsNotMachO" xml:space="preserve">
     <value>image is not a MachO binary</value>
   </data>
+  <data name="MetadataCondition_ImageIsARM64BitBinary" xml:space="preserve">
+    <value>image is an ARM 64 bit binary</value>
```

Comment:
>ARM 64 [](http://example.com/codeflow?start=23&length=6)

ok, so this is a user-facing string and not a .NET API, so we prefer the casing that customers see on the web when querying around for this kind of thing. I think 'ARM64' is best, what do you think? Notice you've introduced a space in your rendering, 'ARM 64', which I propose to eliminate.

Reply:
Is "MetadataCondition_ImageIsARM64BitBinary" good as is? Or does it need to be updated to MetadataCondition_ImageIsArm64BitBinary? I believe I found all of the others.

Reply:
Yes, please update as you suggest. Also, we should probably use 'ARM64 binary' or 'ARM 64-bit binary' in the user-facing string, as each is a bit more readable. I think the latter is good, what do you think?